system/include/emscripten/bind.h

```diff
@@ -1536,6 +1547,7 @@ namespace emscripten {
             .function("size", &MapType::size)
             .function("get", internal::MapAccess<MapType>::get)
             .function("set", internal::MapAccess<MapType>::set)
+            .function("keys", internal::MapAccess<MapType>::keys)
```

Comment:
Hm, one possibly confusing issue here is that, to call this method, a binding for `vector<key_type>` must also be registered. That might be okay if the documentation for this method said so.

If you update the docs, I'm fine with this addition.

Reply:
Added usage example and note about `register_vector` to `site/source/docs/porting/connecting_cpp_and_javascript/embind.rst`